src/presetsController.ts

```diff
@@ -392,18 +411,19 @@ export class PresetsController {
             }
 
             if (newPreset) {
-                const name = await this.showNameInputBox();
+
+                const before: preset.ConfigurePreset[] = await this.getAllConfigurePresets();
+                const name = await this.showNameInputBox() || newPreset.displayName || undefined;
```

Comment:
@gcampbell-msft I thought about having a default name when the user selects a compiler for a new preset, so if they hit escape/don't write a name, it would make the preset with the default name. The current implementation uses the display name as the default name, but I think if there's certain characters cmake doesn't escape correctly(?), so would it be better to just name it "my preset" or something?

Reply:
What if we just use `newPreset.name`? We don't have to use `displayName`, and the `name` should be supported, right?

Reply:
offline discussion summary: the `name` is currently set as "placeholder" so for now we'll set the name as `displayName`